.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: Build
+
+on: push
+
+jobs:
+  patch-src:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: GitHub Environment Variables Action
+        uses: FranzDiebold/github-env-vars-action@v1.2.1
+
+      - name: Set up Debian Docker container for changelog patching
+        # '--ignore-branch' for testing on feature branch
+        run: |
+          docker create --name chglog --volume ${{ github.workspace }}:${{ github.workspace }} --workdir ${{ github.workspace }}/ --tty pitop/gbp-dch-gen:latest sleep inf
+          docker start chglog
+          docker exec chglog git config --global user.name "José Expósito"
+          docker exec chglog git config --global user.email "jose.exposito89@gmail.com"
+
+      - name: Patch changelog (snapshot)
```

Comment:
Where can I see an example of how the log looks like?

Reply:
> Hi! I finally managed to review this, thanks a lot!! I added a few questions, mostly so I can understand what is going on, the PR looks fine.
> 
> I'd be nice to generate builds for DEB amd64 and RPM, but I can add those in the future 😄

You can add amd64 builds easily by specifying the architectures for the `Build Debian package` stage! RPMs are not managed here, sadly!

Reply:
> Where can I see an example of how the log looks like?

We use this workflow in our Python SDK: https://github.com/pi-top/pi-top-Python-SDK/tree/master/.github/workflows
You can see them running in the repo's Actions tab: https://github.com/pi-top/pi-top-Python-SDK/actions

Note that our test workflow does an amd64 build, as this is the host architecture on Github, and so is the fastest for us to build to verify that it will successfully build. We produce ARM builds when we create a GitHub release.

It is easy to see which workflows are which, as you will see things like:
```
Test #870: Commit 5f6a624 pushed by jcapona
```
and
```
Build #445: Commit 7da3e91 pushed by m-roberts
```